src/util.rs

```diff
@@ -187,6 +187,52 @@ pub fn read_encoder_input(kind: &Kind, data: &[u8]) -> Result<StreamDeckInput, S
     }
 }
 
+/// Read inputs from MiraBox E3EN
+pub fn mirabox_e3en_read_input(kind: &Kind, input: u8, state: u8) -> Result<StreamDeckInput, StreamDeckError> {
```

Comment:
These functions could be merged with the ajazz03 ones by adding `1` as a parameter to all of them
This would reduce code duplication, but I wanted to provide the solution with less changes to existng code first

Reply:
Please do that instead, it's less code to review and less code duplication

You can call it mirabox_v2_read_input etc

DRY ("Don't repeat yourself")

Reply:
It's not really `mirabox_v2` but more a `ajazz03_..._v2` by how it is changed as you can see in https://github.com/streamduck-org/elgato-streamdeck/pull/43/commits/d17f573649ed7f21cdee82890fab273758dfcf6b

Reply:
As they are called only when it is a `mirabox_v2` I see why you would replace the names there, but for me it seems not to entirely be a mirabox_v2 feature; maybe it should be a `is_akp03_like()` method or something similar

Reply:
Sure, maybe akp03_family (or, actually, n3_family/n3en_family, to keep all the terminology Mirabox and not Ajazz)

Reply:
Sure, maybe akp03_family (or, actually, n3_family/n3en_family, to keep all the terminology Mirabox and not Ajazz)

Reply:
What you have right now is fine, though, leave it be